custom/icds_reports/ucr/expressions.py

```diff
@@ -322,8 +323,9 @@ class BooleanChoiceQuestion(JsonObject):
 
 @icds_ucr_quickcache(('user_id',))
 def _get_user_location_id(user_id):
-    user = CommCareUser.get(user_id)
-    return user.user_data.get('commcare_location_id')
+    docs = iter_docs(CommCareUser.get_db(), [user_id])
+    doc = next(docs)
```

Comment:
Offlined with @emord but I think this should be `CommCareUser.get_db().get(user_id)`.  Also, is `user_id` guaranteed to exist at this point?  With either implelementation, you may wanna handle that and return `None`

Reply:
If the user id doesn't exist then that means the form was submitted by a user not in the system, which is not expected. In that case it should error and be sent to the pillow retry queue to be retried or investigated, so I'm not going to add error handling